Source: John Kallimanis
Upstream: Will submit
Reason: Without it build is difficult to debug
--- configure.ac	2017-02-22 23:37:55.174315010 +0200
+++ configure.ac3	2017-02-22 23:39:53.473584789 +0200
@@ -25,10 +25,6 @@
    AM_MAKEFLAGS="$AM_MAKEFLAGS --no-print-directory";
 fi
 
-if make --help 2>&1 | grep -q quiet; then
-   AM_MAKEFLAGS="$AM_MAKEFLAGS --quiet"
-fi
-
 if libtool --help 2>&1 | grep -q quiet; then
    AM_LIBTOOLFLAGS="--quiet";
 fi
